admin/load.php

```diff
@@ -634,5 +663,69 @@ function perflab_plugin_admin_notices() {
 		</div>
 		<?php
 	}
+
+	$active_modules_with_plugins = perflab_get_active_modules_with_standalone_plugins();
+
+	if ( ! empty( $active_modules_with_plugins ) ) {
+		$module_data            = perflab_get_modules();
+		$available_modules_list = array();
+		foreach ( $active_modules_with_plugins as $module_slug ) {
+			if ( isset( $module_data[ $module_slug ] ) && ! perflab_is_standalone_plugin_loaded( $module_slug ) ) {
+				$available_modules_list[] = $module_data[ $module_slug ]['name'];
+			}
+		}
+
+		$modules_count = count( $available_modules_list );
+		if ( $modules_count < 1 ) {
+			return;
+		}
+
+		if ( 1 === $modules_count ) {
+			$message  = '<p>';
+			$message .= sprintf(
+				/* translators: Module name */
+				esc_html__( 'Performance Lab has detected usage of "%s" module that have standalone plugin equivalents. Installing and activating a performance plugin will disable the legacy module equivalent. This will not impact functionality of the module.', 'performance-lab' ),
+				esc_attr( $available_modules_list[0] )
+			);
+			$message .= '</p>';
+		} else {
+			$message  = '<p>' . esc_html__( 'Performance Lab has detected usage of modules that have standalone plugin equivalents. Please see the available plugins below. Installing and activating a performance plugin will disable the legacy module equivalent. This will not impact functionality of the module.', 'performance-lab' ) . '</p>';
+			$message .= '<strong>' . esc_html__( 'Available standalone plugins:', 'performance-lab' ) . '</strong>';
+			$message .= '<ol>';
+			foreach ( $available_modules_list as $names ) {
+				$message .= sprintf( '<li>%s</li>', esc_html( $names ) );
+			}
+			$message .= '</ol>';
+		}
+
+		echo '<div class="notice notice-warning is-dismissible">';
+			echo wp_kses_post( $message );
+			echo '<p class="perflab-button-wrapper">';
+				echo '<button type="button" class="button button-primary perflab-install-active-plugin">';
+					echo esc_html__( 'Migrate legacy modules to standalone plugins', 'performance-lab' );
+				echo '</button>';
+				echo '<span class="dashicons dashicons-update hidden"></span>';
+			echo '</p>';
+		echo '</div>';
```

Comment:
As a follow up to my comment in https://github.com/WordPress/performance/pull/899#discussion_r1424435651, here's what I meant with directly printing HTML to get syntax highlighting:
```suggestion
		?>
		<div class="notice notice-warning is-dismissible">
			<?php echo wp_kses_post( $message ); ?>
			<p class="perflab-button-wrapper">
				<button type="button" class="button button-primary perflab-install-active-plugin">
					<?php echo esc_html__( 'Migrate legacy modules to standalone plugins', 'performance-lab' ); ?>
				</button>
				<span class="dashicons dashicons-update hidden"></span>
			</p>
		</div>
		<?php
```
Other than syntax highlighting, it's also a bit more appropriate in terms of indentation, since indenting PHP just because it outputs HTML is a bit odd.